### PR TITLE
v3.5 Changelog

### PR DESCRIPTION
**Pull request by esethna**
Opening WIP PR for the v3.5 changelog.

PR Tracking Spreadsheet:
https://docs.google.com/spreadsheets/d/1hXRBe0dF0rWhaCencjJ8QH4EHmYeOF9bnNSvxWMJUL4/edit#gid=0

### CODE REVIEW

**Comment by esethna** [comment]
@jasonblais take a look at this

| Component  | Tests Run   | Tests Failed                                   |
|:-----------|:------------|:-----------------------------------------------|
| Server     | 948         | :white_check_mark: 0                           |
| Web Client | 123         | :warning: [2 (see details)](http://linktologs) |
| iOS Client | 78          | :warning: [3 (see details)](http://linktologs) |

**Comment by esethna** [comment]
@esethna take a look at this

| Component  | Tests Run   | Tests Failed                                   |
|:-----------|:------------|:-----------------------------------------------|
| Server     | 948         | :white_check_mark: 0                           |
| Web Client | 123         | :warning: [2 (see details)](http://linktologs) |
| iOS Client | 78          | :warning: [3 (see details)](http://linktologs) |

File: source/administration/changelog.md [hunk 1]
```diff
@@ -2,6 +2,248 @@
...
+- Added indexes for `UpdateAt`
+- Added indexes for `DeleteAt`
+- Added full text indexes for `idx_users_all_txt`: Username, FirstName, LastName, Nickname, Email
+- Added full text indexes for `idx_users_names_txt`:Username, FirstName, LastName, Nickname
+
+### API Changes from v3.4 to v3.5
+
+**New routes:**
+
+- Added `GET` at `/users/search`
```
Comment (enahum) [inline source/administration/changelog.md]: this one is `POST`

File: source/administration/changelog.md [hunk 2]
```diff
@@ -2,6 +2,248 @@
...
+    - Search for user profiles based on username, full name and optionally team id.
+- Added `POST` at `/users/{user_id}/update_roles`
+    - Update the users system wide roles.
+- Added `GET` at `/users/{offset}/{limit}`
+    - Retrieves a page of system-wide users
+- Added `POST` at `/teams/{team_id}/update_member_roles`
+    - Update a user's roles for the specified team.
+- Added `GET` at `/teams/{team_id}/channels/{channel_id}/member/{user_id}`
```
Comment (enahum) [inline source/administration/changelog.md]: `/teams/{team_id}/channels/{channel_id}/members/{user_id}`

File: source/administration/changelog.md [hunk 3]
```diff
@@ -2,6 +2,248 @@
...
+    - Retrieves the channel member for the specified user. Useful for fetching the channel member after updates are made to it. If the channel member does not exist, then return an error.
+- Added `GET` at `/teams/{team_id}/stats`
+    - Returns stats for teams
+- Added `POST` at `/teams/{team_id}/members/{offset}/{limit}`
```
Comment (enahum) [inline source/administration/changelog.md]: this one is `GET`

File: source/administration/changelog.md [hunk 4]
```diff
@@ -2,6 +2,248 @@
...
+    - To page through team members
+- Added `GET` at `/teams/{team_id}/members/ids`
```
Comment (enahum) [inline source/administration/changelog.md]: this one is `POST`

File: source/administration/changelog.md [hunk 5]
```diff
@@ -2,6 +2,248 @@
...
+    - Retrieves a list of team members based on user ids
+- Added `GET` at `/teams/{team_id}/members/{user_id}`
+    - Retrieves a single team member
+- Added `GET` at `/teams/{team_id}/posts/{post_id}/get_file_infos`
+    - Retrieves file attachment info for a post
+- Added `GET` at `/channels/{channel_id}/member/{user_id}`
```
Comment (enahum) [inline source/administration/changelog.md]: this one is duplicate, already added above

**Pull request merged**